tests/product/test_collect.py

```diff
@@ -160,7 +163,10 @@ def get_query_id(self, sql, host=None):
         ips = self.cluster.get_ip_address_dict()
         if host is None:
             host = self.cluster.master
-        client = PrestoClient(ips[host], 'root')
+        config_path = os.path.join(REMOTE_CONF_DIR, CONFIG_PROPERTIES)
+        config = self.cluster.exec_cmd_on_host(host, 'cat ' + config_path)
+        user = 'root'
+        client = PrestoClient(ips[host], user, PrestoConfig.from_file(StringIO(config), config_path, host))
```

Comment:
As noted, this seems to be an alternative solution to the one that passed the user and host to the client's config-getting function.  I prefer the other way, but definitely we only need one solution.